server/routes/barcode/contacts/CreateContactByPrisonNumberController.ts

```diff
@@ -35,7 +34,7 @@ export default class CreateContactByPrisonNumberController {
     }
     const view = new CreateContactByPrisonNumberView(
       createNewContactByPrisonNumberForm,
-      filterSupportedPrisons(activePrisons),
+      supportedPrisons,
```

Comment:
Filtering of supported prisons is not done in situ anymore, but is done in the prisonService